gitops/README.md

```diff
@@ -173,6 +173,39 @@ spec:
 
 ```
 
+Below snippet shows the RBAC used to grant any users access to the APIExport.
+
+```
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: apibinding:kubernetes
+rules:
+  - apiGroups:
+      - apis.kcp.dev
+    resources:
+      - apiexports
+    resourceNames:
+      - kubernetes # value of the spec/indentity/secretRef/name field of the APIExport
+    verbs:
+      - bind
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: apibinding:kubernetes:all-users
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: apibinding-kubernetes
```

Comment:
this ClusterRole name doesn't match with `apibinding:kubernetes` defined in ClusterRole.

Reply:
Fixed.